js/cryptocom.js

```diff
@@ -2417,6 +2424,161 @@ module.exports = class cryptocom extends Exchange {
+        const data = this.safeValue (response, 'result');
+        const currencyMap = this.safeValue (data, 'currency_map');
+        return this.parseDepositWithdrawFees (currencyMap, codes, 'full_name');
+    }
+
+    parseDepositWithdrawFee (fee, currency = undefined) {
+        //
+        //    {
+        //        full_name: 'Alchemix',
+        //        default_network: 'ETH',
+        //        network_list: [
+        //          {
+        //            network_id: 'ETH',
+        //            withdrawal_fee: '0.25000000',
+        //            withdraw_enabled: true,
+        //            min_withdrawal_amount: '0.5',
+        //            deposit_enabled: true,
+        //            confirmation_required: '0'
+        //          }
+        //        ]
+        //    }
+        //
+        const networkList = this.safeValue (fee, 'network_list');
+        const networkListLength = networkList.length;
+        const result = {
+            'info': fee,
+            'withdraw': {
+                'fee': undefined,
+                'percentage': undefined,
+            },
+            'deposit': {
+                'fee': undefined,
+                'percentage': undefined,
+            },
+            'networks': {},
+        };
+        if (networkList !== undefined) {
+            for (let i = 0; i < networkListLength; i++) {
+                const networkInfo = networkList[i];
+                const networkId = this.safeString (networkInfo, 'network_id');
+                const currencyCode = this.safeString (currency, 'code');
+                const networkCode = this.networkIdToCode (networkId, currencyCode);
+                result['networks'][networkCode] = {
+                    'deposit': { 'fee': undefined, 'percentage': undefined },
+                    'withdraw': { 'fee': this.safeNumber (networkInfo, 'withdrawal_fee'), 'percentage': false },
+                };
+                if (networkListLength === 1) {
+                    result['withdraw']['fee'] = this.safeNumber (networkInfo, 'withdrawal_fee');
+                    result['withdraw']['percentage'] = false;
+                }
+            }
+        }
+        return result;
+    }
+
+    async fetchTransactionFees (codes = undefined, params = {}) {
+        /**
+         * @method
+         * @name coinex#fetchTransactionFees
+         * @description fetch transaction fees
+         * @see https://exchange-docs.crypto.com/spot/index.html#private-get-currency-networks
+         * @param {[string]|undefined} codes list of unified currency codes
+         * @param {object} params extra parameters specific to the coinex api endpoint
+         * @returns {[object]} a list of [fee structures]{@link https://docs.ccxt.com/en/latest/manual.html#fee-structure}
+         */
+        await this.loadMarkets ();
+        const response = await this.spotPrivatePostPrivateGetCurrencyNetworks (params);
+        console.log (response);
```

Comment:
@AlgoQ, since we're deprecating it, don't know if we need to implement it from scratch now